nstack-kotlin-android/src/main/java/dk/nodes/nstack/kotlin/NStack.kt

```diff
@@ -408,6 +411,8 @@ object NStack {
         }
 
         isInitialized = true
+
+        if (autoAppOpenEnabled.not()) return
         ProcessLifecycleOwner.get().lifecycle.coroutineScope.launchWhenCreated {
             appOpenConsumable = withContext(Dispatchers.IO) { appOpen() }
         }
```

Comment:
Can we move this to a method or make the if contain the ProcessLifecycle stuff. We might introduce a bug down the line with that return

Reply:
Done